killerbee/__init__.py

```diff
@@ -140,9 +175,6 @@ def __init__(self, device: str=None, datasource: str=None, gps: str=None) -> Non
                     if gfccspi and subtype == 0:
                         from .dev_telosb import TELOSB
                         self.driver = TELOSB(self.dev)
-                    elif gfccspi and subtype == 1:
```

Comment:
deprecating apimotev1? should that be in the README for devices or drivers or something if so?

Reply:
Good catch